### DISCUSSION TITLE
Getting started

### DISCUSSION THREAD

**Discussion by LeaTark** [umbraco/Umbraco.Workflow.Issues#30] [Ideas]
I'm having some trouble getting started with Workflow.
Hopefully I've come to the right place.

Does content review work without an approval workflow?
We're a small team and only need reminders to check specific content items.

Emails are not being received.
Does workflow use Umbraco's global smtp configuration?

Not seeing review banner on content.
Approval group members are inherited from editors and I'm a member of editors.

Also, the content review report shows all nodes as overdue when the default period is 0 days, which I now know to be a bug.

**Comment by nathanwoulfe** [comment]
Hi @LeaTark 

Reviews work without an approval flow, but do need permissions set. These are typically inherited from the workflow approvals, but can be configured separately. This is done in Workflow : Content reviews, on the settings content app, using the Content item reviews property. Permissions set here are inherited, so adding configuration to the root node will apply to all descendants.

Workflow uses Umbraco's global SMTP settings, but notifications also need to be enabled (via the notifications app in the Workflow settings section, and separately for content reviews, in the content review settings.

Could you try setting approval group members explicitly rather than inheriting - this will rule out issues with user inheritance.

**Reply by LeaTark** [reply]
Ah yes, looks like an inheritance issue.  I see the banner on content when I'm a direct member of the approval group but not when inherited.

Still haven't received any reminder emails but I am getting submissions from Umbraco Forms in the same environment so I know global SMTP settings are OK.

**Comment by ronaldbarendse** [comment]
I'll close this discussion, since the questions have been answered by Nathan and is also available in our documentation on https://docs.umbraco.com/umbraco-workflow.
